packages/next-app/src/contexts/PoolMetadataContext.tsx

```diff
@@ -55,6 +56,10 @@ export enum StatusLabels {
   "Close",
 }
 
+function hasNull(element: PoolMetadataAttribute) {
+  return Object.values(element).includes(null);
+}
```

Comment:
this could be anything, right?
```suggestion
function hasNull(element: Record<unknown, unknown>) {
  return Object.values(element).includes(null);
}
```

Reply:
yes, I was thinking just about the metadata context